documents/Libraries/stdlib/sx-osl/mx_transformnormalM.inline

```diff
@@ -1 +1 @@
-{{in}} * {{mat}}
+transform({{mat}}, {{in}})
```

Comment:
The transform call in OSL is different depending on vector type (normal, point or vector), so to be safe I think you need to type convert the "in" parameter in the call. e.g.:

`transform({{mat}}, normal({{in}}))`

Reply:
This will not work for the vector4 version though, so you probably need separate versions for vector3 vs. vector4.